xbmc/windowing/win10/WinSystemWin10.cpp

```diff
@@ -344,7 +349,8 @@ bool CWinSystemWin10::ChangeResolution(const RESOLUTION_INFO& res, bool forceCha
 
         if (selected != nullptr)
         {
-          changed = Wait(hdmiInfo.RequestSetCurrentDisplayModeAsync(selected));
+          changed = Wait(hdmiInfo.RequestSetCurrentDisplayModeAsync(
+              selected, needHDR ? HdmiDisplayHdrOption::Eotf2084 : HdmiDisplayHdrOption::None));
```

Comment:
Debug log the attributes of the mode + hdr option? Same for toggle. Up to you, xbox is your turf.